package.json

```diff
@@ -13,7 +13,7 @@
     "detox:android:release": "detox build -c android.emu.release && detox test -R 1 -c android.emu.release",
     "detox:ios": "detox build -c ios.sim.debug && detox test -R 1 -c ios.sim.debug --bail",
     "detox:ios:release": "detox build -c ios.sim.release && detox test -R 1 -c ios.sim.release --bail",
-    "ds": "next dev src/design-system/docs",
+    "ds": "cd src/design-system/docs && yarn dev",
```

Comment:
I chose to isolate the design system docs dependencies into it’s own package to avoid dep pollution here & possibly future proof if we chose to pull it out into it's own repo later